travis.sh

```diff
@@ -141,8 +142,10 @@ my_travis_wait 60 catkin build --no-status --summarize || exit 1
 # Source the new built workspace
 travis_run source install/setup.bash;
 
-# Only run tests on the current repo's packages
-TEST_PKGS=$(catkin_topological_order $CI_SOURCE_PATH --only-names)
+# Choose which packages to run tests on
+echo "Test blacklist: $TEST_BLACKLIST"
+TEST_PKGS=$(catkin_topological_order src --only-names | grep -Fvxf <(echo "$TEST_BLACKLIST" | tr ' ;,' '\n'))
```

Comment:
Aha, that explains. I misunderstood the intention of the original code. I just assumed it was broken. For moveit_kinetic_cpp11, since the list of packages was empty, `travis_run catkin build --no-status --summarize --make-args tests -- $TEST_PKGS` was called without any packages. In that case `catkin build` processes all packages.

But now that I understand what the original is supposed to do, I think the whole addition might not really make sense in the normal case. We only needed it for moveit_kinetic_cpp11 because we couldn't (easily) control the unit tests of `fcl` and `octomap`. But in the normal case the unit tests for specific packages could just be disabled inside that package. It's in the same repository as `.travis.yml` after all.

So, maybe we should just scrap this? Or find a nicer way to work for both the normal case and what you did for moveit_kinetic_cpp11?